src/data/types.ts

```diff
@@ -78,6 +78,15 @@ export type Project = {
      * Link to deployed website
      */
     website?: string;
+
+     /**
+     * If the project is an app, link to the app and type of app. 
+     */
+    app?: {
+      url: string;
+      appType: 'ios' | 'android';
```

Comment:
copying my comment from https://github.com/ubclaunchpad/ubclaunchpad.com/pull/78#discussion_r415226977:

> 

> Note that in general, try to avoid repeating namespaces, for example:
> app.type is already the app's type, but app.appType just adds redundant namespace app again when you can already see it is the app's type

Reply:
fixed, I agree. Missed your earlier comment